test/models/orchestration/dns_test.rb

```diff
@@ -138,7 +138,7 @@ def teardown
     test 'should queue dns update' do
       @host.save!
       @host.queue.clear
-      @host.ip = '1.2.3.4'
+      @host.ip = IPAddr.new(IPAddr.new(@host.ip).to_i + 1, Socket::AF_INET).to_s
```

Comment:
I need to change the host ip to trigger the dns update. The host is managed and this PR requires all managed hosts to have the ip to mach the subnet.  This wasn't the case before, so the test passed. To fix the test, i just calculate the succeeding ip. That seemed to be the easiest way to ensure the ip changes and stays within the subnet.